test/mountain_view/component_test.rb

```diff
@@ -17,15 +17,95 @@ def test_styleguide_stubs
     component = MountainView::Component.new("header")
 
     expected_stub = {
-      id: 1,
-      title: "20 Mountains you didn't know they even existed",
-      subtitle: "Buzzfeed title"
-    }
+                      :meta=>"There is this different classes",
+                      :stubs=>
+                        [
+                          {:id=>1, :title=>"20 Mountains you didn't know they even existed",
+                           :subtitle=>"Buzzfeed title"},
+                          {:id=>2, :title=>"You won't believe what happened to this man at Aspen"}
+                        ]
+                     }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.